app/experimenter/openidc/middleware.py

```diff
@@ -30,7 +30,8 @@ def __call__(self, request):
         except Resolver404:
             pass
 
-        openidc_email = request.META.get(settings.OPENIDC_EMAIL_HEADER, None)
+        default_email = settings.DEV_USER_EMAIL if settings.DEBUG else None
```

Comment:
Coulda done the debug check in settings.py and set DEV_USER_EMAIL to None in there and then just refer to here and use the None check on the following line to catch both.

Reply:
But that's just a thought, this is fine as is.